cv/cv/src/cv_utils/camera_projection.py

```diff
@@ -23,51 +26,60 @@ def __init__(self):
             self.depth_values += [self.depth_map[key]]
         self.depth_values = np.array(self.depth_values)
 
+        camera_info = rospy.wait_for_message('/zed/zed_node/rgb/camera_info', CameraInfo)
```

Comment:
The zed camera topic should be `/zed/zed_node/left/camera_info` by default I believe (since it's a stereo camera). Would be good to subscribe too that.

I also wonder if we could combine the left and right frame for better consistency (theoretically everything in the left viewport should match the right viewport except for points not visible). @Jason-Y000 for vis

Reply:
Should be possible. Think it would be implementing some version of homography to associate between points in the right and left frames

Reply:
Yea, possibly looking into using a sift detector would work